modules/c++/sys/source/OSWin32.cpp

```diff
@@ -100,7 +100,7 @@ sys::Pid_T sys::OSWin32::getProcessId() const
 
 void sys::OSWin32::removeFile(const std::string& pathname) const
 {
-    if (DeleteFile(pathname.c_str()) != true)
+    if (DeleteFile(pathname.c_str()) != TRUE)
```

Comment:
Yes, although the actual file is <minwindef.h>, which gets #included from <windows.h>.

This code might better be tweaked to be "if (!DeleteFile(...))" or "if (DeleteFile(...) == FALSE)" as the docs say "If the function succeeds, the return value is nonzero. If the function fails, the return value is zero (0)."